.zenodo.json

```diff
@@ -0,0 +1,34 @@
+{
+  "creators": [
+    {
+      "affiliation": "810 Labs",
+      "name": "Arsenovic, Alex",
+      "orcid": "0000-0002-8599-5873"
+    },
+    {
+      "affiliation": "University of Cambridge",
+      "name": "Hadfield, Hugo",
+      "orcid": "0000-0003-4318-050X"
+    },
+    {
+      "affiliation": "University of Cambridge",
+      "name": "Wieser, Eric",
+      "orcid": "0000-0003-0412-4978"
+    },
+    {
+      "affiliation": "Enthough",
+      "name": "Kern, Robert"
```

Comment:
@rkern, let us know if you want an orcid in here, we can add it in a future patch.